### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ArneKoeckeritz**
@danielgard @matthiaskmt 
The documentation has a lot of room for improvement.
Markus told me to improve it to avoid a lot of conversation flow with the partners and other engineers.

Please review and comment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,9 @@ All you need to implement this is found here:
 ### Connect to komoot App
 <div style="text-align:center"><img src="assets/BLE-Connect.png" width="673" /></div>
 
-#### Komoot App
-The user activates BLE Connect inside the komoot app settings. When doing so, the app starts advertising the komoot navigation BLE service and tells the user to start pairing via their external BLE device.
+#### Pairing the device
+The user must pair the BLE Device with the Handset OS (Android/ iOS) first.
```
Comment (matthiaskmt) [inline README.md]: @ArneKoeckeritz I'm not sure if I get it right. But on iOS there is no pairing possible on OS level between the phone and the BLE device.

Comment (ToBoehmK) [reply README.md]: @matthiaskmt @ArneKoeckeritz I think the pairing step is just necessary for Android.

Comment (ArneKoeckeritz) [reply README.md]: Okay in this case i would like to write a more OS specific documentation.

**Comment by ArneKoeckeritz** [comment]
@danielgard @matthiaskmt 
I iterated and improved

**Pull request merged**